aws/dms/tasks.yml

```diff
@@ -24,6 +24,9 @@ cron-pii:
 - dashboard.circuit_playground_discount_codes
 - dashboard.projects
 - dashboard.foorm_%
+- dashboard.learning_goal_ai_evaluations
+- dashboard.learning_goal_teacher_evaluations
+- dashboard.learning_goal_ai_evaluation_feedbacks
```

Comment:
Is there a data analysis reason to include the free-form text `other_content` column in this export? It could contain sensitive data.

Reply:
yes. we are planning to analyze these data.